docs/src/user_docs/stdlib/math/u64.md

```diff
@@ -0,0 +1,69 @@
+| wrapping_mul |  Performs multiplication of two unsigned 64 bit integers discarding the overflow.<br /> The input values are assumed to be represented using 32 bit limbs, but this is not checked.<br /> Stack transition looks as follows:<br /> [b_hi, b_lo, a_hi, a_lo, ...] -> [c_hi, c_lo, ...], where c = (a * b) % 2^64 |
+| overflowing_mul |  Performs multiplication of two unsigned 64 bit integers preserving the overflow.<br /> The input values are assumed to be represented using 32 bit limbs, but this is not checked.<br /> Stack transition looks as follows:<br /> [b_hi, b_lo, a_hi, a_lo, ...] -> [c_hi, c_mid_hi, c_mid_lo, c_lo, ...], where c = (a * b) % 2^64<br /> This takes 18 cycles. |
+| checked_mul |  Performs multiplication of two unsigned 64 bit integers, fails when overflowing.<br /> The input values are assumed to be represented using 32 bit limbs, fails if they are not.<br /> Stack transition looks as follows:<br /> [b_hi, b_lo, a_hi, a_lo, ...] -> [c_hi, c_lo, ...], where c = (a * b) % 2^64 |
+| unchecked_div |  Performs division of two unsigned 64 bit integers discarding the remainder.<br /> The input values are assumed to be represented using 32 bit limbs, but this is not checked.<br /> Stack transition looks as follows:<br /> [b_hi, b_lo, a_hi, a_lo, ...] -> [c_hi, c_lo, ...], where c = a // b |
+| checked_div |  Performs division of two unsigned 64 bit integers discarding the remainder.<br /> The input values are assumed to be represented using 32 bit limbs, fails if they are not.<br /> Stack transition looks as follows:<br /> [b_hi, b_lo, a_hi, a_lo, ...] -> [c_hi, c_lo, ...], where c = a // b |
+| unchecked_mod |  Performs modulo operation of two unsigned 64 bit integers.<br /> The input values are assumed to be represented using 32 bit limbs, but this is not checked.<br /> Stack transition looks as follows:<br /> [b_hi, b_lo, a_hi, a_lo, ...] -> [c_hi, c_lo, ...], where c = a % b |
+| checked_mod |  Performs modulo operation of two unsigned 64 bit integers.<br /> The input values are assumed to be represented using 32 bit limbs, fails if they are not.<br /> Stack transition looks as follows:<br /> [b_hi, b_lo, a_hi, a_lo, ...] -> [c_hi, c_lo, ...], where c = a % b |
+| unchecked_divmod |  Performs divmod operation of two unsigned 64 bit integers.<br /> The input values are assumed to be represented using 32 bit limbs, but this is not checked.<br /> Stack transition looks as follows:<br /> [b_hi, b_lo, a_hi, a_lo, ...] -> [r_hi, r_lo, q_hi, q_lo ...], where r = a % b, q = a / b |
+| checked_divmod |  Performs divmod operation of two unsigned 64 bit integers.<br /> The input values are assumed to be represented using 32 bit limbs, fails if they are not.<br /> Stack transition looks as follows:<br /> [b_hi, b_lo, a_hi, a_lo, ...] -> [r_hi, r_lo, q_hi, q_lo ...], where r = a % b, q = a / b |
```

Comment:
nit: for consistency, I would probably use "a // b" here as well instead of "a / b"